README.md

```diff
@@ -0,0 +1,56 @@
+[![Build and Test](https://github.com/open205/libtk205/actions/workflows/build-and-test.yml/badge.svg)](https://github.com/open205/libtk205/actions/workflows/build-and-test.yml)
+
+libtk205
+===========
+
+A C++ library autogenerated from the current version of the ASHRAE Standard 205P data exchange specification.
```

Comment:
We should have a link to toolkit-205 somewhere.

Reply:
Crap, I meant to do that. On it.